### PR TITLE
Check wowza config

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/WowzaApiClient.php [hunk 1]
```diff
@@ -40,4 +40,28 @@ public function setWowzaConfig($wowzaConfig)
     {
         $this->wowzaConfig = $wowzaConfig;
     }
+
+    /**
+     * @param Config $wowzaConfig
+     * @return Integer
+     */
+    public function checkWowzaConfig($wowzaConfig) {
```
Comment (robertszeker) [inline src/WowzaApiClient.php]: can you use type hints here?

Comment (ghost) [reply src/WowzaApiClient.php]: Am i not using type hints? Or where/what do you mean?

File: tests/WowzaApiClientTest.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+        $result = $this->obj->checkWowzaConfig($this->wowzaConfig);
+        $this->assertEquals(404, $result);
+    }
+
+    /**
+     * @test
+     */
+    public function wrongWowzaPass() {
+        $guzzleRequest = $this->prophesize('\GuzzleHttp\Message\Request');
```
Comment (omid) [inline tests/WowzaApiClientTest.php]: It looks like you can create $guzzleRequest and call $this->client->createRequest in the "setUp" function.

File: src/WowzaApiClient.php [hunk 3]
```diff
@@ -40,4 +40,28 @@ public function setWowzaConfig($wowzaConfig)
...
     }
+
+    /**
+     * @param Config $wowzaConfig
+     * @return Integer
+     */
+    public function checkWowzaConfig(Config $wowzaConfig) {
+        $url = $wowzaConfig->getApiUrl() .
+            '/livesetmetadata' .
+            '?app=' . $wowzaConfig->getApp();
```
Comment (robertszeker) [inline src/WowzaApiClient.php]: I would prefer to set the query parameters in a more clean way, by setting them into the $option array in the key 'query', as they did here: http://docs.guzzlephp.org/en/stable/quickstart.html#query-string-parameters.

File: tests/WowzaApiClientTest.php (hunk 2 above)
Comment (ghost) [reply tests/WowzaApiClientTest.php]: I can do that with guzzle request but create request won't work because it needs the function not the return value.

**Review by robertszeker — APPROVED** [review]

**Pull request merged**